packages/tocco-ui/src/EditableValue/typeEditors/TimeEdit.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import DurationEdit from './DurationEdit'
+import {calculateMilliseconds} from '../utils'
+
+const TimeEdit = props => {
+  const propHours = (props.value === null || props.value === undefined) ? 0 : props.value.hoursOfDay
+  const propMinutes = (props.value === null || props.value === undefined) ? 0 : props.value.minutesOfHour
+
+  const millisecondsToDuration = milliSeconds => {
```

Comment:
since "calculateMilliseconds" is move to util, this could also? Better for unit testing as well

Reply:
to that's a no on this comment?

Reply:
Moved it to utils.js.